setup.py

```diff
@@ -25,5 +25,5 @@
     packages=["geosyspy"],
     include_package_data=True,
     data_files=[('', ['VERSION.txt'])],
-    install_requires=["requests", "requests-oauthlib", "oauthlib", "scipy", "pandas==1.3.5", "shapely", "rasterio", "xarray"]
+    install_requires=["requests", "requests-oauthlib", "oauthlib", "scipy", "pandas>=1.3.5", "shapely", "rasterio", "xarray"]
```

Comment:
Why has it been changed ?

Reply:
This change was not meant to be pushed, now, it is canceled!